Lib/test/libregrtest/main.py

```diff
@@ -213,6 +213,13 @@ def parse_args(self, kwargs):
         if ns.tempdir:
             ns.tempdir = os.path.expanduser(ns.tempdir)
 
+        # introduce a way to skip the ref leak tests
+        if ns.huntrleaks:
+            if ns.ignore_tests is None:
+                ns.ignore_tests = ["*skip_ref_leak_test"]
+            else:
+                ns.ignore_tests.append("*skip_ref_leak_test")
```

Comment:
I don't like the implicitness of ignoring tests based on naming. These kinds of tests should be explicitly marked. I'd prefer one of these approaches instead:

1. collect tests that don't support ref-leaks in a dedicated class a la gh-105122
2. add a decorator variant of the solution in gh-105122 (possibly in test.support) and add that to the relevant test(s)

Reply:
After thinking about this, I agree with Eric that we should perhaps not make it easier to skip running tests multiple times (for example, for ref counting). So, let's be explicit like in #104614. Can you please modify this PR accordingly?

Reply:
Sure, I will close that decorator PR then.

Reply:
Thanks!